src/compas_fab/robots/planning_scene.py

```diff
@@ -167,21 +177,24 @@ def remove_collision_mesh(self, id):
         self.robot.client.remove_collision_mesh(id)
 
     def append_collision_mesh(self, collision_mesh, scale=False):
-        """Appends a collision mesh that already exists in the planning scene.
-
-        If the does not exist, it is added.
+        """Append a collision mesh to the planning scene.
 
         Parameters
         ----------
-        collision_mesh : :class:`compas_fab.robots.CollisionMesh`
-            The collision mesh we want to append.
-        scale : bool, optional
-            If `True`, the mesh will be scaled according to the robot's scale
+        collision_mesh : :class:`CollisionMesh`
+            The collision mesh we want to append to the :class:`PlanningScene`.
+        scale : :obj:`bool`, optional
+            If ``True``, the mesh will be scaled according to the robot's scale
             factor.
 
         Returns
         -------
-        None
+        ``None``
+
+        Note
```

Comment:
I would move this note to be  after the first line of the summary, and would rephrase it to explain more in detail how `append` differs from `add`

Reply:
Ok! Reason I keep Notes so far down is because that's the order given in [numpydoc spec](https://numpydoc.readthedocs.io/en/latest/format.html) but I can move them up.

Reply:
I mean, move the text to become part of a multi-line summary text removing the `Note` heading, not that you move the `Note` heading up ;)

Reply:
Please check 9a6dff2. I rephrased both add and append to try to make it clearer.